### ISSUE TITLE
TimeoutError: waiting for selector "button[aria-label="Leave audio"]"

### ISSUE THREAD

**Issue by namanmanish** [jibon57/bbb-recorder#31]
I am trying to use livejoin to record the meeting.
It always give me this error
TimeoutError: waiting for selector "button[aria-label="Leave audio"]" failed: timeout 30000ms exceeded
    at new WaitTask (/home/naman/bbb-recorder/node_modules/puppeteer/lib/DOMWorld.js:549:28)
    at DOMWorld._waitForSelectorOrXPath (/home/naman/bbb-recorder/node_modules/puppeteer/lib/DOMWorld.js:478:22)
    at DOMWorld.waitForSelector (/home/naman/bbb-recorder/node_modules/puppeteer/lib/DOMWorld.js:432:17)
    at Frame.waitForSelector (/home/naman/bbb-recorder/node_modules/puppeteer/lib/FrameManager.js:627:47)
    at Frame.<anonymous> (/home/naman/bbb-recorder/node_modules/puppeteer/lib/helper.js:112:23)
    at Page.waitForSelector (/home/naman/bbb-recorder/node_modules/puppeteer/lib/Page.js:1122:29)
    at main (/home/naman/bbb-recorder/liveJoin.js:78:20)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  name: 'TimeoutError'
}

I tried adjusting the timeout but doesn't seems  to work
Any Ideas?

**Comment by jibon57** [comment]
Make sure that the interface is in English. If you change then it will stop working.
